<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Meduag</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>494</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="spinBox_ts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="suffix">
     <string> m</string>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Session</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>STOP STIMULATION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="pushButton_sp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
border-radius: 40px;</string>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>EXCITABILITY TEST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PW limit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_limit_mA">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe MDL2 Assets</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="suffix">
      <string> mA</string>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_limit_pW">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe MDL2 Assets</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="suffix">
      <string> us</string>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rheobase</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cr">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cronaxie</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current limit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>551</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>STIMULATION PARAMETERS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QSpinBox" name="spinBox_tf_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ramp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_r_1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ramp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T On</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_tn_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ch2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>150</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T Off</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_tf_1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_tn_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_ma_1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="suffix">
      <string> ma</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T Off</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_r_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ch1">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_ma_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="suffix">
      <string> ma</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T On</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="spinBox_f">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> s</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Freq</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_pw">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> us</string>
    </property>
    <property name="minimum">
     <number>300</number>
    </property>
    <property name="maximum">
     <number>700</number>
    </property>
    <property name="singleStep">
     <number>50</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pw</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
